app/models/ExportsDeclaration.scala

```diff
@@ -117,7 +117,8 @@ case class ExportsDeclaration(
   def isNotExs: Boolean = !isExs
 
   def isEntryIntoDeclarantsRecords: Boolean = parties.isEntryIntoDeclarantsRecords.exists(_.answer == YesNoAnswers.yes)
-  def isNotEntryIntoDeclarantsRecords: Boolean = !isEntryIntoDeclarantsRecords
+  def isNotEntryIntoDeclarantsRecords: Boolean = parties.isEntryIntoDeclarantsRecords.exists(_.answer == YesNoAnswers.no)
+  def isNotEntryIntoDeclarantsRecordsOrNoAnswer: Boolean = isNotEntryIntoDeclarantsRecords || parties.isEntryIntoDeclarantsRecords.isEmpty
```

Comment:
Do we really need these 2 `def`? Isn't `isEntryIntoDeclarantsRecords`, the method at 119, enough for our needs?

I mean... do we actually need to differentiate between
```parties.isEntryIntoDeclarantsRecords.isEmpty``` and
```parties.isEntryIntoDeclarantsRecords.exists(_.answer == YesNoAnswers.no)``` ?